packages/lesswrong/server/resolvers/analyticsResolvers.ts

```diff
@@ -199,23 +200,23 @@ addGraphQLResolvers({
       // keys, the partial is no longer partial
       return postAnalytics as PostAnalyticsResult;
     },
-    // TODO make this handle posts as well
-    async AuthorAnalytics(
+    async MultiPostAnalytics(
```

Comment:
Changes to this function:
 - Renamed from AuthorAnalytics to MultiPostAnalytics because it used to just handle a userId, now it can handle that or a list of postIds. Internally it always dealt with a list of posts
 - Added meanReadingTime and uniqueClientViews